css/style.css

```diff
@@ -0,0 +1,157 @@
+
+
+@import url('https://fonts.googleapis.com/css2?family=Heebo&family=PT+Sans&family=Roboto:wght@500&family=Rowdies:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Heebo&family=PT+Sans&display=swap');
+body{
+    margin: 0 auto;
+    background-color: black;
+    color: white;
+}
+.header{
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+}
+.headertitle{
+    font-size: 3rem;
+    font-family: 'Rowdies', cursive;
+    padding-bottom: 3rem;
+    align-self: center;
+}
+.header-sub-title{
+    padding: 1rem;
+    font-size: 2rem;
+    font-family: 'Rowdies', cursive;
+    align-self: center;
+}
+.quiz-card-main{
+    display: flex;
+    justify-content: center;
+    gap:1%;
```

Comment:
use rem or pixel value in the gap. Due to percentage there isn't any row space between cards.

Reply:
Correct the code to use rem, now this is working as expected